src/ray/protobuf/autoscaler.proto

```diff
@@ -138,6 +138,9 @@ message NodeState {
 
   // The instance type name, e.g. p3.2xlarge
   string instance_type_name = 11;
+
+  // The activity of the node.
```

Comment:
can u also comment on some examples?

Reply:
maybe just add the part `descriptive message about why the node is not idle` or something similar.